langchain4j-vertex-ai/src/main/java/dev/langchain4j/model/vertexai/VertexAiRankerScoringModel.java

```diff
@@ -0,0 +1,144 @@
+package dev.langchain4j.model.vertexai;
+
+import com.google.cloud.discoveryengine.v1beta.*;
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.model.output.Response;
+import dev.langchain4j.model.scoring.ScoringModel;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.stream.Collectors;
+
+import static dev.langchain4j.internal.ValidationUtils.ensureNotBlank;
+
+/**
+ * Implementation of a <code>ScoringModel</code> for the Vertex AI Ranking API:
+ * https://cloud.google.com/generative-ai-app-builder/docs/ranking
+ */
+public class VertexAiRankerScoringModel implements ScoringModel {
```

Comment:
```suggestion
public class VertexAiScoringModel implements ScoringModel {
```
Do you ming renaming it?

Reply:
Done. Pushed.